doc/ping.xml

```diff
@@ -357,7 +357,12 @@ xml:id="man.ping">
           <emphasis remap="I">mark</emphasis> to tag the packets
           going out. This is useful for variety of reasons within
           the kernel such as using policy routing to select
-          specific outbound processing.</para>
+          specific outbound processing.
+          CAP_NET_ADMIN capability is required, see
```

Comment:
This should probably also mention CAP_NET_RAW since 5.17?

Reply:
Originally I did not want, but because I updated man page (https://www.alejandro-colomar.es/src/alx/linux/man-pages/man-pages.git/commit/?id=527f31b9f762d0b61aa6aa5da8b8f4e63520b1fd) I'll update it.